pkg/dao/repository_config.go

```diff
@@ -322,6 +324,72 @@ func ModelToApiFields(repoConfig models.RepositoryConfiguration, apiRepo *api.Re
 	}
 }
 
+func convertSortByToSQL(SortBy string) string {
```

Comment:
instead of returning a comma separated string, it could return an array, and then you could do something like (psudo code, not actual go):

```
orderStatements.each do |order|
   filteredDB = filteredDB.Order(order)
end
```

I feel i like thats a little cleaner than a comma separated string: https://gorm.io/docs/query.html#Order

Reply:
oh a pq.StringArray{}?

Reply:
got it.